rust/processor/src/worker.rs

```diff
@@ -631,6 +631,7 @@ impl Worker {
                     .map(|t| timestamp_to_iso(&t))
                     .unwrap_or_default(),
                 end_txn_timestamp_iso = batch_end_txn_timestamp
+                    .clone()
```

Comment:
is this clone necessary?

Reply:
hmm yea the previous one is required; removed now.